packages/playwright-core/src/protocol/protocol.yml

```diff
@@ -833,13 +833,14 @@ BrowserContext:
 
     tracingStopChunk:
       parameters:
-        save: boolean
-        skipCompress: boolean
+        mode:
+          type: enum
+          literals:
+          - doNotSave
+          - compressTrace
+          - compressTraceAndSources
       returns:
         artifact: Artifact?
-        entries:
-          type: array
-          items: NameValue
         sourceEntries:
```

Comment:
Perhaps this should be optional now? We only return it when `mode === compressTrace`, right?